uber/models/department.py

```diff
@@ -407,66 +407,66 @@ def required_roles_ids(self, value):
         self._set_relation_ids('required_roles', DeptRole, value)
 
     @property
-    def hours(self):
-        hours = set()
-        for i in range(self.duration):
-            hours.add(self.start_time + timedelta(hours=i))
-        return hours
+    def minutes(self):
+        minutes = set()
+        for i in range(int(self.duration)):
+            minutes.add(self.start_time + timedelta(minutes=i))
+        return minutes
 
     @property
     def end_time(self):
-        return self.start_time + timedelta(hours=self.duration)
+        return self.start_time + timedelta(minutes=self.duration)
 
     def working_limit_ok(self, attendee):
         """
-        Prevent signing up for too many shifts in a row. `hours_worked` is the
-        number of hours that the attendee is working immediately before plus
-        immediately after this job, plus this job's hours. `working_hour_limit`
-        is the *min* of Department.max_consecutive_hours for all the jobs we've
-        seen (including self). This means that if dept A has a limit of 3 hours,
-        and dept B has a limit of 2 hours, (for one-hour shifts), if we try to
+        Prevent signing up for too many shifts in a row. `minutes_worked` is the
+        number of minutes that the attendee is working immediately before plus
+        immediately after this job, plus this job's minutes. `working_minutes_limit`
+        is the *min* of Department.max_consecutive_minutes for all the jobs we've
+        seen (including self). This means that if dept A has a limit of 3 minutes,
+        and dept B has a limit of 2 minutes, (for one-hour shifts), if we try to
         sign up for the shift order of [A, A, B], B's limits will kick in and
         block the signup.
         """
 
-        attendee_hour_map = attendee.hour_map
```

Comment:
This horrible diff is largely just me find-replacing every instance of `hour` with `minute`. I did also bump the default limit up to 60000 (the old limit multiplied by 60).